### ISSUE TITLE
New quoting mechanism

### ISSUE THREAD

**Issue by sabineschaller** [interledger/open-payments#274]

**Comment by mkurapov** [comment]
@sabineschaller just to double check, for grants, do you think leaving `limits.sendAmount` still makes sense, instead of changing to `maxSendAmount`? I think it makes sense to leave it the same, since we have `limits`. Thoughts?

**Comment by sabineschaller** [comment]
I agree. I think due to the `limits` term, this can remain `sendAmount`
